mk-local-include.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Binutils provide certain headers that are clashing with system-wide
+# headers on Cygwin and MacOS. They do so in different manners, though,
+# so it is not possible to just tweak the inclusion/library order.
+# Hence, we prepare a filtered inclusion directory where we only place
+# the headers that we need for BFD/binutils. To determine them, we'll
```

Comment:
i don't think we need to go through the generation process.  if you look at the `build.sh` script in https://github.com/uclinux-dev/prebuilts-binutils-libs, we have a list of all the headers we care about there.  just copy those over.